forge-gui/res/cardsfolder/upcoming/goblin_negotiation.txt

```diff
@@ -0,0 +1,8 @@
+Name:Goblin Negotiation
+ManaCost:X R R
+Types:Sorcery
+A:SP$ DealDamage | ValidTgts$ Creature | NumDmg$ X | ExcessSVar$ Excess | SubAbility$ DBToken | SpellDescription$ CARDNAME deals X damage to target creature. Create a number of 1/1 red Goblin creature tokens equal to the amount of excess damage dealt to that creature this way.

```

Comment:
`ExcessSVarTargeted$ True`

Reply:
May I just ask what this is and why doesn't Bottle-Cap Blast have it?

Reply:
Well good catch but that one I need to fix with `ExcessSVarCondition$ Permanent` instead

I think I might also remove this extra param and write them as `ExcessSVarCondition$ Card.targetedBy`

Reply:
Well actually the param for the Blast is redundant since excess damage isn't defined for Players anyway